Externals/wxWidgets3/wx/wxmsw.h

```diff
@@ -465,7 +465,7 @@
 //
 // Recommended setting: 1 but may be disabled if you are writing programs
 // running only in C locale anyhow
-#define wxUSE_XLOCALE       1
```

Comment:
I don't have a Windows machine nearby, but I _think_ this is why the Windows build is failing.